private/bufpkg/bufremoteplugin/bufremotepluginconfig/bufremotepluginconfig.go

```diff
@@ -440,6 +441,7 @@ type ExternalGoRegistryConfig struct {
 		Module  string `json:"module,omitempty" yaml:"module,omitempty"`
 		Version string `json:"version,omitempty" yaml:"version,omitempty"`
 	} `json:"deps,omitempty" yaml:"deps,omitempty"`
+	BasePluginName string `json:"base_plugin_name,omitempty" yaml:"base_plugin_name,omitempty"`
```

Comment:
```suggestion
	BasePlugin string `json:"base_plugin,omitempty" yaml:"base_plugin,omitempty"`
```

This might be clearer and more concise. The intended format is remote/owner/plugin

Reply:
Changed in a2edfb7f to "base plugin" terminology.